packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -31,28 +30,25 @@ class FormAds extends Component {
 
     componentDidMount() {
         // TODO: [p2p-fix-api] call get offer detail api and populate state
-        WS().send({ 'residence_list': 1 }).then(() => { // this is just to mock the api delay response
-            const new_initial_values = {
-                country        : 'Indonesia',
-                currency       : 'IDR',
-                type           : 'buy',
-                asset          : 'USD',
-                fix_price      : 10000,
-                amount         : 50,
-                min_transaction: 1000,
-            };
-            this.setState({
-                initial_values: new_initial_values,
-            });
-
-            if (this.props.ad_id) {
-                // call the api, get the file based on id
-                // populate the state from the respnose
-            } else {
-                this.setState({ is_loading: false });
-            }
-
+        const new_initial_values = {
```

Comment:
this file is not related to this PR, since we are not releasing the form of my ads, IMO we shouldn't make changes to have unrelated git diff. but up to you.

Reply:
you're right, but its okay, just want to pollute the code less :)